src/workspace.coffee

```diff
@@ -550,7 +556,7 @@ class Workspace extends Model
   # Extended: Create a new text editor.
   #
   # Returns a {TextEditor}.
-  buildTextEditor: (params) ->
+  buildTextEditor_: (params) ->
```

Comment:
I don't think you should *need* to suffix this with an underscore because this version is on the prototype whereas the version bound in the constructor will be on the instance. Maybe that's too vague though.